package.json

```diff
@@ -70,7 +70,8 @@
     "prettier": "3.2.4",
     "protobufjs-cli": "1.1.2",
     "ts-node": "9.1.1",
-    "typescript": "4.1.2"
+    "typescript": "5.6.2",
+    "@noble/curves": "1.6.0"
```

Comment:
Only as "dev-dependency" for now (no change of production code).

By the way, a lot happened since https://github.com/multiversx/mx-deprecated-sdk-js-wallet/pull/51 (on top of v1.6.0):
 - https://github.com/paulmillr/noble-curves/releases